Example/MoyaComparison/MoyaComparison/Services/ServicesClients/MoyaVideoWebService.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2017 niji
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import RxSwift
+import Moya
+
+struct MoyaVideoWebService: VideoWebService {
+  var webServices: WebServices
+  
+  func fetchAllVideos() -> Observable<[Video]> {
+    return self.webServices.customAPIProvider.request(CustomAPI.getVideos()).mapJSON()
+      .flatMap({ (json: Any) -> Observable<[Video]> in
+        return self.webServices.serializationSwiftyJSON.parse(objects: json)
+      })
+  }
+  
+  func fetchVideo(identifier: Int) -> Observable<Video> {
+    return self.webServices.customAPIProvider.request(CustomAPI.getVideo(identifier: identifier)).mapJSON()
+      .flatMap({ (json: Any) -> Observable<Video> in
+        return self.webServices.serializationSwiftyJSON.parse(object: json)
+      })
+  }
+  
+  func updateVideo(video: Video) -> Observable<Void> {
```

Comment:
Swift 3 conventions. Avoid repeating the `video` name twice, because at call site it will have to be written `updateVideo(video: video)` which is quite repeating.

Use either `func update(video: Video)` (call site will be `update(video: video)`) or `func updateVideo(_ video: Video)` (call site will be `updateVideo(video)`. I personally prefer the first one.